packages/ui/components/form/select/components.tsx

```diff
@@ -80,7 +80,7 @@ export const ControlComponent = <
     {...props}
     className={classNames(
       className,
-      "dark:bg-darkgray-100 dark:border-darkgray-300 border-gray-300 bg-white text-sm leading-4 placeholder:text-sm placeholder:font-normal focus-within:border-0 focus-within:ring-2 focus-within:ring-neutral-800 hover:border-neutral-400 dark:focus-within:ring-white"
+      "dark:bg-darkgray-100 dark:border-darkgray-300 !min-h-9 h-9 border-gray-300 bg-white text-sm leading-4 placeholder:text-sm placeholder:font-normal focus-within:border-0 focus-within:ring-2 focus-within:ring-neutral-800 hover:border-neutral-400 dark:focus-within:ring-white"
```

Comment:
Turns out the default height of the react-select element is `38px`, so we are overriding the height here
Here's a reference [link](https://stackoverflow.com/questions/54218351/changing-height-of-react-select-component#:~:text=all%20of%20bootstraps%20stuff%20is,which%20looks%20a%20little%20odd.)